stable/aws-pod-identity-webhook/templates/service.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "aws-pod-identity-webhook.fullname" . }}
+  labels:
+{{ include "aws-pod-identity-webhook.labels" . | indent 4 }}
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    prometheus.io/port: "443"
+    prometheus.io/scheme: "https"
+    prometheus.io/scrape: "true"
+spec:
```

Comment:
Did you want to templatize the Service's type?

Example: https://github.com/aws/eks-charts/blob/575321fdf9d68a656c75768805cc80175b92275e/stable/appmesh-prometheus/templates/service.yaml#L8

Reply:
Let's add this in a later PR